lib/StateApi.js

```diff
@@ -1,10 +1,14 @@
 class StateApi {
 	
-	constructor(rawData){
+	constructor(rawData, dateString){

```

Comment:
for dealing with invalid checksum as new StateApi call is on two places 
- [Server renderr](https://github.com/Mhtshum/AdvancedReactSB/blob/005b0b04a40188c43604439e389a867603efb2f6/lib/renderers/server.js#L13l)
- [Dom render](https://github.com/Mhtshum/AdvancedReactSB/blob/005b0b04a40188c43604439e389a867603efb2f6/lib/renderers/dom.js#L6)